user-guide/manual.adoc

```diff
@@ -115,22 +119,107 @@ TBD
 |BSL Source[[bsl-source]]
 |https://github.com/NASA-AMMOS/BSL[bsl-source]
 
-|BSL Document Source[[bsl-docs]]
+|BSL Documentation Source[[bsl-docs]]
 |https://github.com/NASA-AMMOS/BSL-docs[bsl-docs]
 
+|Bundle Protocol Version 7[[RFC9171]]
+|https://www.rfc-editor.org/info/rfc9171
```

Comment:
Minor nit: We can align the formatting for references in the two docs either now or later.